sample-lnd.conf

```diff
@@ -1275,6 +1276,20 @@ litecoin.node=ltcd
 ; Otherwise errors may occur in lnd under high-load conditions.
 ; db.postgres.maxconnections=
 
+[sqlite]
+; The path to the sqlite db.
+; db.sqlite.dbpath=/path/lnd.db
```

Comment:
The DB path is actually a directory. Maybe we should make that more explicit and call it `dbdir` instead?